libs/fetcher.js

```diff
@@ -93,7 +93,12 @@ function parseParamValues (params) {
             var request;
 
             if (req.method === GET) {
-                var path = req.path.substr(req.path.lastIndexOf('/resource/') + '/resource/'.length).split(';');
```

Comment:
Since there will always be a leading slash, this can just be :

```js
var path = req.path.substr(1).split(';'); // strip leading slash with substring(1)
```